plugin/wavesurfer.regions.js

```diff
@@ -37,12 +37,17 @@ WaveSurfer.Regions = {
         var start;
         var region;
 
-        this.wrapper.addEventListener('mousedown', function (e) {
+        function eventDown(e) {
             drag = true;
+            if(typeof e.targetTouches != 'undefined'){
```

Comment:
I think we also need to check if it's a single touch, otherwise we would prevent pinch zoom and other multitouch gestures from working properly.